services/discoverysrv/service.go

```diff
@@ -0,0 +1,52 @@
+package discoverysrv
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/ethereum/go-ethereum/common"
+	common3 "github.com/iden3/go-iden3/common"
+	"github.com/iden3/go-iden3/utils"
+)
+
+type Trusted struct {
+	Relay bool `json:"relay"`
+}
+
+type Identity struct {
```

Comment:
Maybe `Entity` is a better definition since it involves `IdAddr` isnside and it could be a misunderstanding among `Identity` and `IdAddr`

Reply:
Changed in the following commit :)

Reply:
Nice !! :top: